ui/src/components/DownloadButton/DownloadButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import Button from '@splunk/react-ui/Button';
+
+interface DownloadButtonProps {
+    // needs to be same domain if not it will just open link
+    fileUrl: string;
+    fileNameAfterDownload: string;
+    children: ReactElement | ReactElement[] | string;
+}
+
+function DownloadButton(props: DownloadButtonProps) {
+    return (
+        <Button
+            target="_blank"
```

Comment:
```suggestion
            openInNewContext
```

Reply:
does it make sense to create a new tab that is automatically closed?

Reply:
the problem is that when you add this `openInNewContext ` the icon at the end is added by default

![Screenshot 2023-11-08 at 12 14 53](https://github.com/splunk/addonfactory-ucc-generator/assets/143183665/26711fd1-bfe7-4750-96b5-343076736e42)

also the `target="_blank"` is useful as when url is not from the same domain it is opened in another tab instead of current one you can try it by using ie. this link `https://dummyjson.com/products` in storybook